src/main/java/Bert/Bert.java

```diff
@@ -0,0 +1,357 @@
+package Bert;
+
+import exceptions.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Bert {
+    private static ArrayList<Task> taskAL = new ArrayList<Task>();
+    public static void main(String[] args) {
+        String goodbyeMessage = "\tBye. Hope to see you again soon!";
+        welcomeMenu();
+        try{
+            readFromFile();
+        } catch (IOException e){
+            println("READERROR");
+        }
+        Scanner in = new Scanner(System.in);
+        while(true){
+            String line = in.nextLine();
+            String command = commandCheck(line);
+            switch (command){
+                case "bye":
+                    println(goodbyeMessage);
+                    try{
+                        writeToFile();
+                    } catch(IOException e){
+                        println("IO: smth wrong");
+                    }
+                    return;
+                case "mark":
+                    try{
+                        markTask(line);
+                    } catch (MarkUnmarkNumberError e) {
+                        println("\tERROR(Mark): Number Missing");
+                    } catch (MarkUnmarkItemError e) {
+                        println("\tERROR(Mark): Item does not exist");
+                    }
+                    break;
+                case "unmark":
+                    try{
+                        unmarkTask(line);
+                    } catch (MarkUnmarkNumberError e) {
+                        println("\tERROR(Unmark): Number Missing");
+                    } catch (MarkUnmarkItemError e) {
+                        println("\tERROR(Unmark): Item does not exist");
+                    }
+                    break;
+                case "list":
+                    listTasks();
+                    break;
+                case "todo":
+                case "deadline":
+                case "event":
+                    addTask(line);
+                    break;
+                case "delete":
+                    try{
+                        deleteTask(line);
+                    } catch (DeleteNumberError e){
+                        println("\tERROR(Delete): Number Missing");
+                    } catch (DeleteItemError e){
+                        println("\tERROR(Delete): Item does not exist");
+                    }
+                    break;
+                default:
+                    println("\tERROR: Invalid command");
+                    break;
+            }
+        }
+    }
+
+
+
+    private static String cleanFrontSpacing(String line){
+        return line.replaceFirst("^\\s*", "");
+    }
+    private static String commandCheck(String line) {
```

Comment:
- naming doesn't follow our convention for method names.
- can use some vertical space for better readability